### PR TITLE
Implemented basic outline for frontend

### LINKED ISSUE #5: Setup basic frontend
Will start working on it once #1 is complete(by merging #4)

### PR DESCRIPTION
**Pull request by gannonprudhomme**
Includes set up for:

- npm - our package manager
- webpack - compiles react & typescript into javascript for the browser to interpret
- typescript
- react

Closes #5

### CODE REVIEW

**Comment by rconn478** [comment]
Consider moving README from `autoscheduler/frontend/src` to `autoscheduler/frontend`, and reset settings.py back to blank username/password

**Comment by gannonprudhomme** [comment]
> Consider moving README from `autoscheduler/frontend/src` to `autoscheduler/frontend` so it's easier to find

I agree with moving the README back from source, just will need to make a note in the README for the install/setup steps to have `src/` be your current working directory.

> and reset settings.py back to blank username/password
Definitely an accident, will fix

File: autoscheduler/autoscheduler/settings.py [hunk 1]
```diff
@@ -79,8 +81,8 @@
         'NAME': 'dbautoscheduler',
         'HOST': 'localhost',
         'PORT': '5432',
-        'USER': '',
-        'PASSWORD': '',
+        'USER': 'postgres',
```
Comment (gannonprudhomme) [inline autoscheduler/autoscheduler/settings.py]:
```suggestion
        'USER': '',
```

**Comment by gannonprudhomme** [comment]
Removed settings.py database user/pass & moved frontend to frontend/ folder

**Pull request merged**